<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>impedancechart</class>
 <widget class="QWidget" name="impedancechart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="ConnectQualityWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>340</width>
       <height>340</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="Fp1label">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>45</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Fp1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Fp2label">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>45</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Fp2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="F7label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>94</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>F7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="F3label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>F3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="F4label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>F4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="F8label">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>94</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>F8</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="T3label">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>164</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>T3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="T4label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>164</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>T4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="A1label">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>164</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>A1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="A2label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>164</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>A2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="T5label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>T5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="T6label">
      <property name="geometry">
       <rect>
        <x>263</x>
        <y>250</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>T6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="P3label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>P3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="P4label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>P4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="O1label">
      <property name="geometry">
       <rect>
        <x>113</x>
        <y>300</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>O1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="O2label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>300</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>O2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Fzlabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Fz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="C3label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>159</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>C3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="C4label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>159</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>C4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Czlabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>159</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Cz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Pzlabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>220</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Pz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="X1label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>X1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="X2label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>X2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="X3label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>X3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
